app/assets/javascripts/components/forms/timezoneDropdown.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PureComponent } from 'react';
+import { Field } from 'redux-form';
+import TimezonePicker from 'react-bootstrap-timezone-picker';
+import _ from 'lodash';
+
+const TIMEZONES = {
+  '(GMT-08:00) America/Los_Angeles': 'America/Los_Angeles',
+  '(GMT-07:00) America/Denver': 'America/Denver',
+  '(GMT-06:00) America/Chicago': 'America/Chicago',
+  '(GMT-05:00) America/New_York': 'America/New_York',
+};
+
+class TimezoneDropdown extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
+    this.findTimezoneLabel = this.findTimezoneLabel.bind(this);
```

Comment:
This doesn't need to be bound here. You only need to bind functions if you're passing them down to other components like how you're passing the `onChange` function `TimezonePicker`.